src/features/PortfolioTable.js

```diff
@@ -1,35 +1,62 @@
-import React, {useState} from "react";
+import React, {useEffect, useState} from "react";
 import '../PortfolioPage.css';
+import axios from "axios";
 
 function PortfolioTable(props) {
-    const [portfolio, setPortfolio] = useState(props.portfolio);
+    const [portfolio, setPortfolio] = useState([]);
     const [sharesToBuyOrSell, setSharesToBuyOrSell] = useState({});
+    const [sharePrices, setSharePrices] = useState({});
+
+    const API_KEY = "cghh1b9r01qjd0395910cghh1b9r01qjd039591g"
```

Comment:
API key is here, might be an issue? I figure since this is a dead app soon, it won't matter too much